wasm2c/examples/rot13/main.c

```diff
@@ -78,6 +83,8 @@ int main(int argc, char** argv) {
   /* Free the Wasm runtime state. */
   wasm_rt_free();
 
+  printf("Done!\n");
```

Comment:
Not sure we need this if we're always either printing the output or a "usage" message...

Reply:
Removed!